### PR TITLE
Retire arcalog

### PR DESCRIPTION
**Pull request by dustinblack**
## Changes introduced with this PR

Removing the arcalog docs from the navigation, and using the existing mdx plugin to embed the markdown from the main github org profile into the docs main page to reduce maintenance overhead.

---
By contributing to this repository, I agree to the [contribution guidelines](https://github.com/arcalot/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

**Review by dbutenhof — APPROVED** [review]
Well, OK, that certainly looks plausible.

I've only seen this `{! ... !}` syntax once, and it referenced a particular set of lines. More importantly, I was never able to find any real documentation on it. Maybe I just haven't guessed the right search keys. And ... is there a way to test this stuff locally? There's a simple markdown preview in `vscode`, but it doesn't handle "advanced stuff".

**Comment by dustinblack** [comment]
> Well, OK, that certainly looks plausible.
> 
> I've only seen this `{! ... !}` syntax once, and it referenced a particular set of lines. More importantly, I was never able to find any real documentation on it. Maybe I just haven't guessed the right search keys. And ... is there a way to test this stuff locally? There's a simple markdown preview in `vscode`, but it doesn't handle "advanced stuff".

The mdx plugin was added previously for other markdown inclusions by @mfleader. Docs here: https://github.com/neurobin/mdx_include

To test locally, you just need to install the dependencies from the `requirements.txt` file and then run `mkdocs serve` and you can see the docs page rendered locally.

File: docs/index.md [hunk 1]
```diff
@@ -4,62 +4,4 @@ hide:
...
----
-
-## Community
-
-You can find our general community health files like our code of conduct and contribution guidelines in the [.github repository](https://github.com/arcalot/.github). If you have any questions or suggestions, please use the issues in the respective repository or contribute to the [discussions](https://github.com/orgs/arcalot/discussions).
-
-If you would like to contribute, check out the issues in the individual repositories and our [project boards](https://github.com/orgs/arcalot/projects) where we organize our work.
-
-If you want to participate in our bi-weekly meeting you can add our recurring meeting to your calendar: [Arcalot Project Community meeting (iCal)](meetings/arcalot-community-biweekly.ics)
+{! https://raw.githubusercontent.com/arcalot/.github/main/profile/README.md !}
```
Comment (jaredoconnell) [inline docs/index.md:7]: It may make sense to add a comment to the (non-raw) github page for that file for ease of use.

Comment (jaredoconnell) [reply docs/index.md:7]: Just to point to where the referenced file is.

**Review by webbnh — COMMENTED** [review]
This seems fine, assuming the change to remove Arcalog from the _included_ content is coming under separate cover.

However, running `mkdocs`, it issues the following:
```
The following pages exist in the docs directory, but are not included in the "nav" configuration:
             - arcaflow/workflows/schemas.md
             - arcalog/index.md
```
Should this PR also be removing the Arcalog assets from the repo?  (They will always be available in the historical commits....)

And, if you're game, it looks like `schemas.md` should be added under
```
      - Writing workflows:
```

**Comment by dustinblack** [comment]
> This seems fine, assuming the change to remove Arcalog from the _included_ content is coming under separate cover.
> 
> However, running `mkdocs`, it issues the following:
> 
> ```
> The following pages exist in the docs directory, but are not included in the "nav" configuration:
>              - arcaflow/workflows/schemas.md
>              - arcalog/index.md
> ```
> 
> Should this PR also be removing the Arcalog assets from the repo? (They will always be available in the historical commits....)
> 
> And, if you're game, it looks like `schemas.md` should be added under
> 
> ```
>       - Writing workflows:
> ```

I intentionally left the arcalog docs and just removed the navigation and the main page references. That way any existing hyperlinks will continue to work. It seemed like the best way to "archive" the content.

**Comment by dustinblack** [comment]
> ```
> The following pages exist in the docs directory, but are not included in the "nav" configuration:
>              - arcaflow/workflows/schemas.md
>              - arcalog/index.md
> ```

It looks like `schemas.md` was added by @mfleader in PR #81 but not added to the navigation.

**Pull request merged**